appveyor.yml

```diff
@@ -0,0 +1,50 @@
+build: false
+
+environment:
+  matrix:
+    - PYTHON_VERSION: 2.7
+      PYTHON_SVER: 27
+      MINICONDA: C:\Miniconda
+    - PYTHON_VERSION: 3.4
+      PYTHON_SVER: 34
+      MINICONDA: C:\Miniconda3
+
+init:
+  - "ECHO %PYTHON_VERSION% %MINICONDA%"
+
+install:
+  # Issues have been encountered with installing numpy and scipy on
+  # AppVeyor e.g.
+  # http://tjelvarolsson.com/blog/how-to-continuously-test-your-python-code-on-windows-using-appveyor/
+  # Miniconda is recommended as the way to install these. See also:
+  # https://github.com/appveyor/ci/issues/359
+  # The following adopts approaches suggested in the above links.
+  - "set PATH=%MINICONDA%;%MINICONDA%\\Scripts;%PATH%"
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+  - conda info -a
+  - "conda create -q -n test-environment python=%PYTHON_VERSION%"
+  - activate test-environment
+
+  # Majority of dependencies can be installed with Anaconda
+  - conda install numpy scipy matplotlib sympy networkx nose h5py pandas
+
+  # Graphviz & PyGraphviz
+  - appveyor DownloadFile "http://graphviz.org/pub/graphviz/stable/windows/graphviz-2.38.zip" -FileName graphviz.zip
+  - 7z x graphviz.zip -y > nul
+  - set "PATH=%cd%\release\lib\release\dll;%PATH%"
+  - pip install https://dl.dropboxusercontent.com/u/6251352/pygraphviz-1.3.1-cp%PYTHON_SVER%-none-win32.whl?dl=1
```

Comment:
I found a conda channel offering pygraphviz 1.3.1 for win32/win64: `marufr`. So `conda install -c marufr pygraphviz` ought to do it, then we won't need to maintain this in dropbox.